index.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
+
+function setCase(str, strCase) {
+	return str['to' + strCase + 'Case']();
+}
+
 module.exports = function (str, plural, count) {
 	if (typeof plural === 'number') {
 		count = plural;
-		plural = str + 's';
+
+		var plur = 's';
+		// get the tail of the string to check if the case should be upper or lower
+		var strCase = str.slice(1) === str.slice(1).toUpperCase() ? 'Upper' : 'Lower';
```

Comment:
```js
'foobar'.slice(1);
//=> "oobar"
```

It would make sense to only grab the last character. We really only care what the last character is as that's the character we want to match. Imagine a cameCased word: `camelCs`. We only want to match the lowercase `s`.